### ISSUE TITLE
Getting started instructions do not provide clear reason/explanation

### ISSUE THREAD

**Issue by nmindz** [spacelift-io/terraform-starter#20]
This is something **really minor** but that still blocked me as I do not understand the implications.

At the [README](https://github.com/spacelift-io/terraform-starter#so-you-want-to-take-spacelift-for-a-spin) file there's a bold instruction to **"... not to change the repository's name when creating your copy"**.

I wanted to fork this template and bring it under my private org so I can tweak it and re-use it as a template for my new project repositories (this is the purpose of this repository, am I correct?). However we have a slightly different naming convention that I would most like to keep, which in turn involves changing the repository name.

Personally I can't seem to find an obvious risk/issue with doing that, except for that disclaimer. Is there something important that I'm unaware tied to the repository name within itself?

If possible, please explain at the README doc _why_ people should not change the repository name after copying/forking. That would help a lot and make much more sense than a flat warning.

**Comment by jmfontaine** [comment]
@nmindz I am the one who added this months ago but I recall that something broke down the road because I had renamed the repository but I cannot remember what. I will try and see if/why it breaks.

**Comment by nmindz** [comment]
Thanks a lot, @jmfontaine! I appreciate the effort of looking into it. 😃

**Comment by jmfontaine** [comment]
@nmindz I found why 😃 

The repository name is hardcoded in the [Spacelift stack definition](https://github.com/jmfontaine/my-terraform-starter/blob/main/stack.tf#L7).

I hear you though. Would it be better if I replaced that recommendation by a warning to update that name if the repository is given a different name?

**Comment by nmindz** [comment]
@jmfontaine That and an indication to where it should be changed would be superb! 😀

**Comment by jmfontaine** [comment]
Yes, of course. I will indicate where it needs to be changed … or I could make it a game with clues. 😄
